generate-templates.yaml

```diff
@@ -170,6 +176,7 @@
       icon: fedora
       oslabels: "{{ fedora_labels }}"
       osinfoname: "{{ oslabels[0] }}"
+      cloudusername: fedora
```

Comment:
Why RHELs have `cloudusername: cloud-user` but others have `cloudusername` their name?

Reply:
`cloud-user` is the default user name for RHEL
The purpose here is to specify the default usernames to better UX in the console